packages/deployment-service/cdk/lib/cdk-stack.ts

```diff
@@ -188,6 +189,28 @@ export const createStack = async () => {
 
   env['CODEBUILD_PIPELINE_UPDATE_TOPIC_ARN'] = codebuildSnsTopic.topicArn
 
+  const authorizerLambda = createFunction(
+    stack,
+    'deployment-service-authorizer-lambda',
+    path.resolve(__dirname, '..', '..', 'dist', 'authorizer'),
+    'authorizer/index.handler',
+    {
+      ISSUERS: config.ISSUERS,
+      CLIENT_ID: config.COGNITO_CLIENT_ID,
+      CONNECT_USER_POOL: config.CONNECT_USER_POOL,
+    },
+    undefined,
+    undefined,
```

Comment:
Your default memory allocation is 1024Mb which I'd say for node is way too high. I'd consider explicitly setting this here to something lower (can't remember which undefined param it is). You'll more than likely see the same performance with 512